src/server/api/routers/webhook.ts

```diff
@@ -20,30 +22,35 @@ export const webhookRouter = createTRPCRouter({
         name: 'web',
         active: true,
         events: [
-          'push',
           "issues",
           'pull_request'
         ],
         config: {
           url: WEBHOOK_URL,
           content_type: 'json',
-          insecure_ssl: '0'
+          insecure_ssl: '0',
+          secret: env.WEBHOOK_SECRET
         },
         headers: {
           'X-GitHub-Api-Version': '2022-11-28'
         }
       })
 
       if (response.status === 201) {
-        console.log("Webhook created successfully", response);
+        // response.headers.location example - location: 'https://api.github.com/repos/GabrielPedroza/exotica/hooks/425484562'
+        const locationURL = response.headers.location!
```

Comment:
this will throw an exception if it's null. if that's possible, then it should be handled

Reply:
This is a concern. I've attempted to look for more definitive information on when the location should be received and nothing concrete surfaced. Location is *imperative* because it is the only way to be able to identify the WebHook. Without this, it would arise issues. My personal assumption is if I utilize a HTTP method, like POST request to GitHub in this case, I would receive the location that it was made in. On the other hand, if I used another method, like GET, I would not receive the location in the headers.